bin/cyhy-ip

```diff
@@ -6,26 +6,27 @@ Usage:
   cyhy-ip [--section SECTION] [--file FILENAME] (check | normalize) [ADDRESSES ...]
   cyhy-ip [--section SECTION] [--file FILENAME] setstage STAGE [ADDRESSES ...]
   cyhy-ip [--section SECTION] list OWNER
-  cyhy-ip [--section SECTION] list-all 
+  cyhy-ip [--section SECTION] list-all
   cyhy-ip [--section SECTION] [--file FILENAME] move OWNER NEW_OWNER [ADDRESSES ...]
   cyhy-ip (-h | --help)
   cyhy-ip --version
 
 Options:
   -h --help                      Show this screen.
   --version                      Show version.
-  
+
   -f FILENAME --file=FILENAME    Read addresses from a file.
   -s SECTION --section=SECTION   Configuration section to use.
-  
+
 Notes:
   Address can be read from standard input if no addresses are provided on the command line
   and the --file option is omitted.
 """
 
+from collections import defaultdict
```

Comment:
The last thing is if you would please organize the imports in this file to mirror that in `bin/cyhy-import`. It would be very, very appreciated.

Reply:
@mcdonnnj = Detective Columbo :smile: 
![image](https://user-images.githubusercontent.com/5521725/121550951-363f3700-c9dd-11eb-9e2d-d345ed7d9648.png)

Reply:
Just pushed an update with the import changes, @mcdonnnj. Sorry it took a while, it's been a hectic week. Thanks!